pkg/model/estimator/local/ratio_process.go

```diff
@@ -29,6 +29,27 @@ import (
 	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
 )
 
+func getNodeTotalResourceUsage(containerMetrics *collector_metric.ContainerMetrics, usageMetric string) float64 {
+	var nodeTotalResourceUsage float64
+	if _, ok := containerMetrics.CounterStats[usageMetric]; ok {
+		nodeTotalResourceUsage = float64(containerMetrics.CounterStats[usageMetric].Delta)
+	} else {
+		// this will be system container (all processes's CPU Time
+		nodeTotalResourceUsage = float64(containerMetrics.CPUTime.Delta)
+	}
+	return nodeTotalResourceUsage
+}
+
+func getProcessResUsage(process *collector_metric.ProcessMetrics, usageMetric string) float64 {
+	var processResUsage float64
+	if _, ok := process.CounterStats[usageMetric]; ok {
+		processResUsage = float64(process.CounterStats[usageMetric].Delta)
+	} else {
+		processResUsage = float64(process.CPUTime.Delta)
```

Comment:
figure out some issue,... config.DRAMUsageMetric can't be used here as we need use different value than CPUTime..

Reply:
Cgroup metrics do not include specific measurements related to memory access. While CPU utilization (i.e. CPUTime) is not directly indicative of memory access, it is currently the only available proxy for such information that we can rely on.

Can you please include a comment about it?